demosplan/DemosPlanCoreBundle/Logic/Report/ExportReportService.php

```diff
@@ -177,7 +179,7 @@ private function writeReportEntry(ReportEntry $reportEntry, Table $table)
 
         $userName = $reportEntry->getUserName();
         $userCell = $table->addCell($this->styles['userCellWidth']);
-        $userCell->addText($userName, $this->styles['baseFont']);
+        $userCell->addText(u($userName)->normalize()->toString(), $this->styles['baseFont']);
```

Comment:
The extra u seems strange.

Reply:
I try with another methods to support umlaut alphabet, but it doesn't work, only with `u()` you can have a correct export